super-mode-calculator/src/lib/date.ts

```diff
@@ -0,0 +1,17 @@
+import { format } from 'date-fns';
+
+export function toDateString(date: Date): string {
+	return format(date, 'yyyy-MM-dd');
+}
+
+export function toDatePath(date: Date): string {
+	return format(date, 'yyyy/MM/dd/');
+}
+
+export function toDateHourString(date: Date): string {
+	return format(date, 'yyyy-MM-dd HH:00:00.000');
+}
+
+export function toDateHourPath(date: Date): string {
+	return format(date, 'yyyy/MM/dd/HH/');
+}
```

Comment:
Have you double checked this is safe w/r daylight savings switchover? I can't see `toISOString`, `.000` or `Z` anywhere!

Reply:
I believe this is safe. All this code runs on lambdas, which should have their local timezones = UTC. However, it might be worth explicitly converting the `date` argument here to UTC before formatting. This makes that assumption explicit, and also corrects the behaviour when running locally.

Reply:
I'll look into doing that now